vcsutils/webhookparser/bitbucketserver.go

```diff
@@ -128,22 +128,50 @@ func (webhook *bitbucketServerWebhookParser) getRepositoryDetails(repository bit
 	}
 }
 
+// Please checkout official documentation about the pull request webhooks at:
```

Comment:
At where? And how this and the other new links are helping us? We could add links for every method, but it would uglify the code.

Reply:
Having a direct link the doc would help. It's a pain looking for it each time we need it.

Reply:
The link are removed now.